src/test/java/seedu/address/logic/commands/ClearCommandTest.java

```diff
@@ -22,7 +22,7 @@ public void execute_emptyAddressBook_failure() {
     }
 
     @Test
-    public void execute_nonEmptyAddressBook_success() {
+    public void execute_emptyAddressBook_throwCommandException() {
```

Comment:
In line 30, is the `assertCommandSuccess(new ClearCommand(), ...)` method call supposed to be changed to `assertCommandFailure(new ClearCommand(), model, String.format(Messages.MESSAGE_EMPTY_CLIENT_LIST, "clear")`?

Reply:
Resolved, thanks for pointing it out. I was supposed to delete this test case and replace it with execute_nonEmptyAddresBook_success() but forgot to do so.